crml/sylo/src/inbox.rs

```diff
@@ -69,8 +69,8 @@ decl_module! {
 
 decl_storage! {
 	trait Store for Module<T: Trait> as SyloInbox {
-		NextIndexes: map hasher(blake2_128_concat) T::AccountId => MessageId;
-		Values get(values): map hasher(blake2_128_concat) T::AccountId => Vec<(MessageId, Message)>;
+		pub NextIndexes: map hasher(blake2_128_concat) T::AccountId => MessageId;
```

Comment:
@cowboy-bebug I highly recommend that we don't make the internal structure of modules public for the sake of the migration task. I'm going to make a PR that would show the alternative which is much easier to be removed later and doesn't harm the encapsulation of the modules.

Reply:
Nice I'll update the script once that's available